fractal-puppy/Engine
# MenuScene and HUD fixed

This pr add the scenes for main menu and HUD with all the images put in their correct place.
To test this is easy:
- Start the engine and load MenuScene or HUD. 
- Check if you see the correct images and not black images everywhere.
